src/libexpr/eval.hh

```diff
@@ -162,7 +162,7 @@ struct DebugTrace {
 };
 
 // Don't want Windows function
-#undef SearchPath
+#undef LookupPath
```

Comment:
This can be removed, including the comment, which I can't put into a GitHub suggestion.

Reply:
Thanks for finding this! Done